examples/helloworld/greeter_server/main.go

```diff
@@ -47,7 +48,9 @@ func (s *server) SayHello(ctx context.Context, in *pb.HelloRequest) (*pb.HelloRe
 
 func main() {
 	flag.Parse()
-	lis, err := net.Listen("tcp", fmt.Sprintf(":%d", *port))
+	// Setting KeepAlive to -1 allows the listener to retain OS default TCP keepalive time and interval.
```

Comment:
```suggestion
	// Setting KeepAlive to -1 allows the listener to retain OS default TCP
        // keepalive time and interval.
```

Reply:
Sorry, but can we revert this file for now?  I know it was mentioned in the issue, but I'm not sure it makes sense to complicate the basic "hello world" with such a detail.

Reply:
@dfawley @arvindbr8 is the consensus to revert this file? I can certainly do so if confirmed! :)